.github/workflows/test-latest-products.yml

```
@@ -0,0 +1,80 @@
+
+name: Test Latest Products
+on:
+  push:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  load-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      supported: ${{ steps.load-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - name: Load Supported Product Build Matrix
+        id: load-matrix
+        run: echo "matrix=$(tr '\n' ' ' < supported_builds.json | tr -s ' ' )" >> "$GITHUB_OUTPUT"
```

If I'm reading this right, this slurps in a json file and removes newlines? What do you think of leveraging jq's compact mode to do the work?

```suggestion
        run: echo "matrix=$(jq --compact-output < supported_builds.json)" >> "$GITHUB_OUTPUT"
```